docs/area-owners.md

```diff
@@ -4,30 +4,29 @@ The below table lists all the `area-`labels used in the **dotnet/aspnetcore** re
+| **[area-minimal](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-minimal)**                   | [@captainsafia](https://github.com/captainsafia) [@mitchdenny](https://github.com/mitchdenny) | [@adityamandaleeka](https://github.com/adityamandaleeka) |                         |
+| **[area-mvc](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-mvc)**                           | [@captainsafia](https://github.com/captainsafia)                                              | [@mkArtakMSFT](https://github.com/mkArtakMSFT)           |                         |
+| **[area-networking](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-networking)**             | [@amcasey](https://github.com/amcasey)                                                        | [@adityamandaleeka](https://github.com/adityamandaleeka) |                         |
+| **[area-perf](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-perf)**                         | [@sebastienros](https://github.com/sebastienros)                                              |                                                          |                         |
+| **[area-razor.compiler](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-razor.compiler)**     |                                                                                               |                                                          |                         |
+| **[area-razor.tooling](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-razor.tooling)**       |                                                                                               |                                                          |                         |
+| **[area-routing](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-routing)**                   |                                                                                               | [@mkArtakMSFT](https://github.com/mkArtakMSFT)           |                         |
+| **[area-security](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-security)**                 |                                                                                               | [@blowdart](https://github.com/blowdart)                 |                         |
+| **[area-signalr](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-signalr)**                   | [@BrennanConroy](https://github.com/BrennanConroy)                                            | [@adityamandaleeka](https://github.com/adityamandaleeka) | Home repo: https://github.com/SignalR/SignalR/ |
```

Comment:
SignalR/SignalR is the repo for classic (non-core) SignalR. I don't think there's value in linking that from here. cc @BrennanConroy

Reply:
Is there a better link or should I just drop it?